src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -4253,6 +4253,12 @@ .FORWARDHELPCATEGORY Cmdlet
     [switch]
     ${ShowWindow})
 
+    # On non-Window systems always display the full help topic.
+    # On Windows, only display the full help topic if the user has specified a PAGER app.
+    if (($psCmdlet.ParameterSetName -eq 'AllUsersView') -and !$Full -and (!$IsWindows -or (Test-Path env:PAGER))) {
+        $PSBoundParameters['Full'] = $true
+    }
```

Comment:
Can we add a test which check the view that gets selected. It should have `#FullView`
```PowerShell
(Get-Help Get-Command -Full).psobject.typenames
```

Reply:
Yes, I can add that.

Reply:
BTW, to be clear, this tweak affects only the `help` function on Linux/macOS/Windows as well as the `man` alias on Windows.  That function pipes through a pager so we don't get rich objects out - just text.  That is why I'm using `-cmatch` to look for certain words that indicate it is full help.